common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/AbstractRootConcept.scala

```diff
@@ -15,6 +21,57 @@ object Concept {
   }
 }
 
+case class DateRange(label: String,
+                     start: LocalDateTime,
+                     end: LocalDateTime,
+                     inferred: Boolean)
+    extends AbstractConcept
+object DateRange {
```

Comment:
I would name this something separate (e.g. `DateRangeParsers`), pull it into a separate file and emphasise that it's a distinct thing that we can test really rigorously.

Reply:
Yea, I was wondering if the standard we followed was keeping things like parsing / sanitising / validating etc with the actual case class themselves (I quite like that, but it can be easy to overload).

Should we be calling the companion objects for the other type `ThingyTypeNormaliser` then?

Or maybe instead of parse, I could use `fromString`, but that is parsing... ah naming.

Reply:
Also - might think about DateRange, as it is `DateTime` first of all.